src/d3d9/d3d9_device.h

```diff
@@ -1564,10 +1568,14 @@ namespace dxvk {
 
     bool                            m_isSWVP;
     bool                            m_isD3D8Compatible;
-    bool                            m_amdATOC          = false;
-    bool                            m_nvATOC           = false;
     bool                            m_ffZTest          = false;
-    
+
+    bool                            m_depthBiasEnabled = false;
```

Comment:
Please add a comment in the code explaining why we have those fields when the helper functions exist.